src/Widgets/DownloadsButton.vala

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright © 2019 Adrian Cochrane (https://adrian.geek.nz)
```

Comment:
Not sure how you feel about copyrights, but I believe we should both be here since the original design and UI were from my prototype. Not super picky, just want it to be as correct as possible. :)

```suggestion
 * Copyright © 2019 Adrian Cochrane (https://adrian.geek.nz)
 *             2019 Cassidy James Blaede (https://cassidyjames.com)
```